bucket/zoom.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "4.6.7",
+    "description": "Video and audio conferencing, chat, and webinars",
+    "homepage": "https://zoom.com.cn/",
+    "license": {
+        "identifier": "Shareware",
+        "url": "https://zoom.com.cn/terms"
+    },
+    "url": "https://zoom.com.cn/client/latest/ZoomInstaller.exe#/dl.7z",
+    "hash": "9c73bc037b0082185978711e4a26701ca9efe69db0dbbfd0292682399c140f7b",
+    "installer": {
+        "script": [
+            "Expand-7zipArchive \"$dir\\Zoom.msi\" \"$dir\"",
```

Comment:
Why 7z?

Reply:
Failed to open zoom.msi using lessmsi, no log. Maybe I'm using this tool in the wrong way, I'm not sure.